plugin.js

```diff
@@ -34,6 +34,12 @@ function fastifyCookieSetCookie (reply, name, value, options, signer) {
 
 function fastifyCookieClearCookie (reply, name, options) {
   const opts = Object.assign({ expires: new Date(1), path: '/' }, options || {})
+  if (opts.signed) {
```

Comment:
How about simply
```js
const opts = Object.assign({
  path: '/'
}, options || {}, {
  expires: new Date(1),
  signed: undefined,
  maxAge: undefined
})
```

I think there's another issue, if `options` contain `expires` then our set `expires` is overwritten and it's not really a "clear" operation.

In the above code, I've moved expires after options.